### DISCUSSION TITLE
OIDC connector support with private_key_jwt auth method

### DISCUSSION THREAD

**Discussion by michalvanco** [dexidp/dex#2140] [Q&A]
Hi guys!

Is there a possibility to integrate a connector in dex which uses the `private_key_jwt` authentication method?

Current version and documentation seems to cover only the options with client secret authentication and we would like to integrate with a provider which allows only the PKCE approach and has:
```
"token_endpoint_auth_methods_supported": [
  "private_key_jwt"
],
```

Thank you in advance for any hints.

**Comment by nabokihms** [comment]
Unfortunately, Dex only supports two other authentication methods - client_secret_basic and client_secret_post. It is not a simple task to add the new authentication option to the current connector config, but we can open a feature request for it.
